keras_hub/src/layers/modeling/anchor_generator.py

```diff
@@ -5,19 +5,14 @@
 
 from keras_hub.src.api_export import keras_hub_export
 
-# TODO: https://github.com/keras-team/keras-hub/issues/1965
-from keras_hub.src.bounding_box.converters import convert_format
-
 
 @keras_hub_export("keras_hub.layers.AnchorGenerator")
 class AnchorGenerator(keras.layers.Layer):
     """Generates anchor boxes for object detection tasks.
-
```

Comment:
these look like copy paste bugs where selecting from github removes empty newlines. bring these empty newlines back

Reply:
Sure

Reply:
This is unfixed. Take a look at the diff here and restore the empty newlines.

If you copy paste from the github ui you will remove all empty newlines from the file, just something to watch out for.